File: src/main/java/com/catalog/catalog/dto/StudentDTO.java
package com.catalog.catalog.dto;

import lombok.Data;

import java.util.List;

@Data
public class StudentDTO {
    private int studentId;
    private String firstName;
    private String lastName;
    private String date;
    private String studentGender;
    private String studentEmail;
    private List<CourseDTO> courses;


}

File: angular-catalog/src/app/class/student-dto.ts
import {CourseDTO} from "./course-dto";

export class StudentDTO {
  studentId: number | undefined;
  firstName: string | undefined;
  lastName: string | undefined;
  date: string| undefined;
  studentGender:string| undefined;
  studentEmail:string| undefined;
  courses:CourseDTO[]| undefined;


}
